cmd/icingadb/main.go

```diff
@@ -253,3 +258,42 @@ func run() int {
 		cancelHactx()
 	}
 }
+
+// checkDbSchema asserts the database schema of the expected version being present.
+func checkDbSchema(ctx context.Context, db *icingadb.DB, dbName string) error {
+	{
+		var one []struct{ One uint8 }
```

Comment:
I wouldn't check if the schema is installed. The `SELECT` will fail anyway if there is no table` icingadb_schema`.

Reply:
Hm.. clever **and** smart :)